### PR TITLE
Improved external API example

### PR DESCRIPTION
**Pull request by floelhoeffel**
- added screenshot
- using DOM methods 
- autosize for iframe

### CODE REVIEW

File: samples/external-api/app.html [hunk 1]
```diff
@@ -1,43 +1,56 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8"/>
+
+<head>
+    <meta charset="UTF-8" />
```
Comment (jelz) [inline samples/external-api/app.html]: No need for self-closing tag `/>`

File: samples/external-api/app.html [hunk 2]
```diff
@@ -1,43 +1,56 @@
...
-        });
-
-        $("#user-id").on('input', function() {
-            dd.setValue(this.value);
-        });
-      })
+        window.contentfulExtension.init(function (api) {
+            api.window.startAutoResizer();
+            var value = api.field.getValue()
+            var dd = api.field;
```
Comment (jelz) [inline samples/external-api/app.html]: `dd`? Either name it `field` or always use `api.field` which is also quite ok for me.

File: samples/external-api/app.html [hunk 3]
```diff
@@ -1,43 +1,56 @@
...
+            var serviceUri = "https://jsonplaceholder.typicode.com/users";
+
+            $.ajax({
+                url: serviceUri,
+                type: "GET",
+                crossDomain: true,
+                dataType: "json",
+                success: function (response) {
+                    $.each(response, function (index, value) {
```
Comment (jelz) [inline samples/external-api/app.html]: `response.forEach(function () { ... })` would be better

File: samples/external-api/app.html [hunk 4]
```diff
@@ -1,43 +1,56 @@
...
+
+                        var option = document.createElement("option")
+                        option.setAttribute("value", value.id)
```
Comment (jelz) [inline samples/external-api/app.html]: Use or don't use `;` but be consistent

File: samples/external-api/app.html [hunk 5]
```diff
@@ -1,43 +1,56 @@
...
+                        option.innerText = value.name
+
+                        $("#user-id").append(option);
+                    });
+                    $("#user-id").val(dd.getValue());
+                },
+                error: function (xhr, status) {
+                    console.log("error");
```
Comment (jelz) [inline samples/external-api/app.html]: `error fetching data`. You could also include status code and response.

File: samples/external-api/app.html (hunk 2 above)
Comment (axe312ger) [reply samples/external-api/app.html]: 👍

**Comment by floelhoeffel** [comment]
Implemented your suggestions - thanks @jelz

**Pull request merged**